src/components/shared/StackImplementationSetting.js

```diff
@@ -47,19 +47,26 @@ class StackImplementationSettingImpl extends PureComponent<Props> {
     labelL10nId: string,
     implementationFilter: ImplementationFilter
   ) {
+    const htmlId = `implementation-radio-${implementationFilter}`;
     return (
-      <label className="photon-label photon-label-micro photon-label-horiz-padding">
-        <input
-          type="radio"
-          className="photon-radio photon-radio-micro"
-          value={implementationFilter}
-          name="stack-settings-filter"
-          title="Filter stack frames to a type."
-          onChange={this._onImplementationFilterChange}
-          checked={this.props.implementationFilter === implementationFilter}
-        />
-        <Localized id={labelL10nId}></Localized>
-      </label>
+      <>
+        <Localized id={labelL10nId} attrs={{ title: true }}>
```

Comment:
note: I'm using the same l10nId for both the label and the input. It looks like either Fluent or the browser ignores the body for inputs.

Reply:
Hm that's interesting, but works for me.